### ISSUE TITLE
Set minimum range

### ISSUE THREAD

**Issue by thalisvilela** [oli107/material-range-bar#168]
Is there a way to define a minimum range? Preventing the user from selecting a smaller range than what is set?

**Comment by krazykira** [comment]
Nope, It is not possible to configure a range once Rangebar is created and then dynamically bound the user within that range

**Comment by Mukarrim** [comment]
Actually it is possible please check my other answer: https://github.com/oli107/material-range-bar/issues/172#issuecomment-535051056
